contracts/main/price-feeders/DelayFathomOraclePriceFeed.sol

```diff
@@ -8,29 +8,27 @@ import "../interfaces/IFathomOracle.sol";
 import "../interfaces/IAccessControlConfig.sol";
 
 contract DelayFathomOraclePriceFeed is PausableUpgradeable, IFathomOraclePriceFeed {
-    struct Feed {
-        uint256 price;
-        uint256 lastUpdateTS;
-    }
-
-    Feed public currentPrice;
+    uint256 public latestPrice;
+    uint256 public delayedPrice;
+    uint256 public lastUpdateTS;
     uint256 public timeDelay;
+    uint256 public priceLife;
 
     IFathomOracle public fathomOracle;
     IAccessControlConfig public accessControlConfig;
     address public token0;
     address public token1;
-    uint256 public priceLife; // [seconds] how old the price is considered stale, default 1 day
 
     function initialize(address _fathomOracle, address _token0, address _token1, address _accessControlConfig) external initializer {
         PausableUpgradeable.__Pausable_init();
 
+        require(_accessControlConfig != address(0), "FathomOraclePriceFeed: ZERO_ADDRESS");
```

Comment:
2.3.4 Unsafe initialize of accessControl in DelayFathomOraclePriceFeed - fixed issue by adding validation for zero addres